minitwit/.drone.yml

```diff
@@ -98,18 +98,12 @@ steps:
       from_secret: root_password
     POSTGRES_PASSWORD:
       from_secret: POSTGRES_PASSWORD
-    WEB_HOST_1:
-      "164.92.150.233"
-    WEB_HOST_2:
-      "206.189.13.233"
+    WEB_HOST:
+      "164.92.150.233 206.189.13.233"
   commands:
   - apk add --no-cache sshpass openssh
-  - sshpass -p "$ROOT_PASSWORD" scp -o StrictHostKeyChecking=no terraform/files/update_image.sh root@$WEB_HOST_1:/tmp/
-  - sshpass -p "$ROOT_PASSWORD" ssh -o StrictHostKeyChecking=no root@$WEB_HOST_1 chmod +x /tmp/update_image.sh
-  - sshpass -p "$ROOT_PASSWORD" ssh -o StrictHostKeyChecking=no root@$WEB_HOST_1 /tmp/update_image.sh registry.digitalocean.com/minitwit/minitwit-server $DRONE_BUILD_NUMBER $DOCKER_USERNAME $DOCKER_PASSWORD $POSTGRES_PASSWORD
-  - sshpass -p "$ROOT_PASSWORD" scp -o StrictHostKeyChecking=no terraform/files/update_image.sh root@$WEB_HOST_2:/tmp/
-  - sshpass -p "$ROOT_PASSWORD" ssh -o StrictHostKeyChecking=no root@$WEB_HOST_2 chmod +x /tmp/update_image.sh
-  - sshpass -p "$ROOT_PASSWORD" ssh -o StrictHostKeyChecking=no root@$WEB_HOST_2 /tmp/update_image.sh registry.digitalocean.com/minitwit/minitwit-server $DRONE_BUILD_NUMBER $DOCKER_USERNAME $DOCKER_PASSWORD $POSTGRES_PASSWORD
+  - chmod +x ./terraform/files/deploy.sh
+  - ./terraform/files/deploy.sh $ROOT_PASSWORD $WEB_HOSTS $DRONE_BUILD_NUMBER $DOCKER_USERNAME $DOCKER_PASSWORD $POSTGRES_PASSWORD
```

Comment:
You called it WEB_HOSTS in the command, but the variable is called WEB_HOST :P

Reply:
Oops good catch :D